promet/nclex
# NCSBNSB-11:  Reorganize Contrib module file structure

This PR moves all of the contrib modules into the "modules/contrib"
directory of the project.

This PR also removes the temporary _info.php file from the drupal
root.

To test:

Navigate to the file structure on the server, and go to the "modules"
directory.  You should only see three directories:

* contrib
* custom

All contrib modules will now reside in "modules/contrib".

Navigate to the drupal root directory; _info.php should no longer be
present in this directory.

branch NCSBNSB-11-reorganize-contrib-module-file-structure
